Fixes to the documentation

yulric opened the pull request:
This PR fixes some issues in the documentation. Recommend going through each commit to see what each change is. In summary:

1. Fixed missing quotes in odm-how-to qmd fikle
2. Updated all parts sheet to have a `firstRelease` and `lastUpdated` columns
3. Updated the `invalid_category` rule specification to allow missing values in a categorical column
4. Made the file names consistent for the `invalid_type` rule assets

@zargot I may need your help with some of your commits appearing in this PR. They're not in the dev branch so I assume they were deleted in a rebase. Should I just rebase dev?

zargot commented:
> This PR fixes some issues in the documentation. Recommend going through each commit to see what each change is. In summary:
> 
> 1. Fixed missing quotes in odm-how-to qmd fikle
> 2. Updated all parts sheet to have a `firstRelease` and `lastUpdated` columns
> 3. Updated the `invalid_category` rule specification to allow missing values in a categorical column
> 4. Made the file names consistent for the `invalid_type` rule assets
> 
> @zargot I may need your help with some of your commits appearing in this PR. They're not in the dev branch so I assume they were deleted in a rebase. Should I just rebase dev?

@yulric, yes, you can simply rebase your 4 commits onto dev, discarding the others.
This can be done in a few ways:
- `git rebase --onto dev HEAD~4`
- `git rebase -i dev`, then delete the lines for the other commits.

yulric commented:
> > This PR fixes some issues in the documentation. Recommend going through each commit to see what each change is. In summary:
> > 
> > 1. Fixed missing quotes in odm-how-to qmd fikle
> > 2. Updated all parts sheet to have a `firstRelease` and `lastUpdated` columns
> > 3. Updated the `invalid_category` rule specification to allow missing values in a categorical column
> > 4. Made the file names consistent for the `invalid_type` rule assets
> > 
> > @zargot I may need your help with some of your commits appearing in this PR. They're not in the dev branch so I assume they were deleted in a rebase. Should I just rebase dev?
> 
> @yulric, yes, you can simply rebase your 4 commits onto dev, discarding the others. This can be done in a few ways:
> 
> * `git rebase --onto dev HEAD~4`
> * `git rebase -i dev`, then delete the lines for the other commits.

Thanks! Should be good for you to review now.

zargot reviewed (approved):
looks good :)
